godot-core/src/registry/callbacks.rs

```diff
@@ -96,7 +109,16 @@ where
     let base_class_name = T::Base::class_name();
     let base_ptr = unsafe { interface_fn!(classdb_construct_object)(base_class_name.string_sys()) };
 
-    match create_rust_part_for_existing_godot_part(make_user_instance, base_ptr) {
+    #[cfg(before_api = "4.4")]
+    let postinit = None;
+    #[cfg(since_api = "4.4")]
+    let postinit = if notify_postinitialize {
+        Some(notify_postinit as unsafe fn(sys::GDExtensionObjectPtr))
```

Comment:
Please avoid the dangerous `as` casts, especially between function pointers.

If there's a type-inference problem, then `as` is not the right way to solve it.